### PR TITLE
do not expose stderr method via os_env

### PR DESCRIPTION
**Pull request by chris-rock**

### CODE REVIEW

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

File: lib/resources/os_env.rb [hunk 1]
```diff
@@ -27,10 +27,6 @@ def split
     @content.split(':', -1)
   end
 
-  def stderr
-    @command_result.stderr
-  end
-
   def exit_status
```
Comment (arlimus) [inline lib/resources/os_env.rb]: This too ;)

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by arlimus** [comment]
@delivery approve

**Comment by chef-delivery** [comment]
Change: 2a868e9f-90a3-4cc6-9c4c-ba510f43ac11 approved by: @arlimus

**Comment by arlimus** [comment]
Great simplification!

**Comment by chef-delivery** [comment]
This PR has passed 'Acceptance' and is ready to be delivered!
  Use: '@delivery deliver' when validated in acceptance.

**Pull request merged**